osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -265,9 +265,14 @@ public virtual void Save(BeatmapInfo info, IBeatmap beatmapContent, ISkin beatma
 
                     // grab the original file (or create a new one if not found).
                     var fileInfo = setInfo.Files.SingleOrDefault(f => string.Equals(f.Filename, beatmapInfo.Path, StringComparison.OrdinalIgnoreCase)) ?? new BeatmapSetFileInfo();
-
+                    
```

Comment:
This shouldn't be in the diff, please format your files with your code editor before committing/making a pull request.